packages/components/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@terra-ui-packages/components",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "build": "tsc && tsc -p tsconfig.cjs.json",
+    "test": "jest"
+  },
+  "type": "module",
+  "module": "./lib/es/index.js",
+  "main": "./lib/cjs/index.js",
+  "exports": {
+    ".": {
+      "import": "./lib/es/index.js",
+      "require": "./lib/cjs/index.js"
+    }
+  },
```

Comment:
Configure exports for using this package via `import` (ES Modules) or `require` (CommonJS). The package API is the same either way, but they resolve to different files.